### PR TITLE
Docker swarm tutorial

### PR DESCRIPTION
**Pull request by yuce**
## Overview

[Describe what this pull request addresses.]

Fixes #

## Pull request checklist

- [ ] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [ ] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [ ] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: docs/tutorials.md [hunk 1]
```diff
@@ -240,7 +240,11 @@ Check out our [Administration Guide](https://www.pilosa.com/docs/latest/administ
 
 ### Setting Up a Docker Cluster
```
Comment (codysoyland) [inline docs/tutorials.md]: Add "Setting Up a Docker Cluster" to the nav.

File: docs/tutorials.md [hunk 2]
```diff
@@ -312,11 +316,82 @@ networks:
   pilosanet:
 ```
```
Comment (codysoyland) [inline docs/tutorials.md]: Make indentation consistent (choose 4 or 2 spaces).

File: docs/tutorials.md [hunk 3]
```diff
@@ -312,11 +316,82 @@ networks:
...
 
+#### Running a Docker Swarm
+
+It is very easy to run a Pilosa Cluster on different servers using [Docker Swarm mode](https://docs.docker.com/engine/swarm/). All we have to do is creating an overlay network instead of the bridge network.
+
+The instructions in this section require Docker 17.06 and better. Although it is possible to run a Docker swarm on MacOS or Windows, it is easiest to run it on Linux. So we assume you are trying these instructions on Linux, probably on the cloud.
+
+We are going to use two servers: the master node runs in the first server and a  slave node in the second server.
```
Comment (codysoyland) [inline docs/tutorials.md]: Use "manager" and "worker" instead of "master" and "slave".

**Comment by yuce** [comment]
Updated.

**Comment by codysoyland** [comment]
I think you missed some changes present in the other PR (d1153755e98cc64fddc9bb1869a0dfc108c398fc).

**Comment by yuce** [comment]
@codysoyland Sorry, I forgot to push the commit before. It's done.

**Pull request merged**